lib/host/tests/index.ts

```diff
@@ -0,0 +1,12 @@
+import {Host} from "../src";
+import * as loader from "../../loader/src";
+import * as fs from "fs";
+import ASModule from "./build";
+import * as asc from "../../../cli/asc"
+type testMod = typeof ASModule;
+
+
+var wasm = fs.readFileSync(`${__dirname}/build/untouched.wasm`);
+let imports = (new Host()).getImports<testMod>();
```

Comment:
Now the imports object contains a special property `post`, which is a list of identity functions that are repeatably applied to the instance by the loader.

Reply:
This was removed.